P5/Source/Specs/post.xml

```diff
@@ -0,0 +1,242 @@
+    contribution to a CMC interaction which has been composed
+    (or recorded) by its author in its entirety before being
+    transmitted over a network (e.g., the internet) and made available
+    on the monitor or screen of the other parties en bloc.</desc>
+    <classes>
+        <memberOf key="model.cmc"/>
+        <memberOf key="att.global"/>
+        <memberOf key="att.cmc"/>
+        <memberOf key="att.indentation"/>
+        <memberOf key="att.ascribed"/>
+        <memberOf key="att.datable"/>
+	<memberOf key="att.fragmentable"/>
+        <memberOf key="att.docStatus"/>
+        <memberOf key="att.timed"/>
+        <memberOf key="att.typed"/>
+        <memberOf key="att.canonical"/>
+    </classes>
+    <content>
+        <alternate minOccurs="0" maxOccurs="unbounded">
+            <textNode/>
+            <classRef key="model.headLike"/>
+            <elementRef key="opener"/>
+            <classRef key="model.pLike"/>
+            <classRef key="model.gLike"/>
+            <classRef key="model.phrase"/>
+            <classRef key="model.inter"/>
+            <classRef key="model.global"/>
+            <elementRef key="lg"/>
+            <classRef key="model.lLike"/>
+            <classRef key="model.divBottom"/>
+        </alternate>
+    </content>
+    <attList>
+        <attDef ident="modality" usage="rec" mode="add">
+            <desc xml:lang="en" versionDate="2021-09-07">written or spoken mode</desc>
+            <datatype minOccurs="1" maxOccurs="1">
+                <dataRef key="teidata.enumerated"/>
+            </datatype>
+            <valList xml:lang="en" type="semi">
+                <valItem ident="written">
+                    <desc xml:lang="en" versionDate="2021-09-07">for stretches of text and/or multimodal elements</desc>
+                </valItem>
+                <valItem ident="spoken">
+                    <gloss xml:lang="en" versionDate="2024-06-30">for audio (or audio-visual) posts</gloss>
+                </valItem>
+            </valList>
+            <remarks xml:lang="en" versionDate="2021-09-07">
+                <p>This attribute is intended to convey information about the CMC environment in
+                    which the post was created, as opposed to the MIME type of the content conveyed,
+                    which is indicated by the elements contained within the <gi>post</gi>.</p>
+            </remarks>
+        </attDef>
+        <attDef ident="replyTo" mode="add">
+            <desc xml:lang="en" versionDate="2021-09-07">indicates to which previous post the
+                current post replies or refers to. This attribute should be used to encode
+                "technical" reply information, i.e. which is due to a formal reply action (such as
+                activating a "reply" button in the client software) and which is also formally
+                represented in the source, e.g. in the "references" field of a Usenet message header
```

Comment:
```suggestion
                represented in the source, e.g. in the <q>references</q> field of a Usenet message header
```

Reply:
I'm applying this and a few other edits for clarity in the post spec.